control/server.py

```diff
@@ -764,6 +779,18 @@ def gateway_rpc_caller(self, requests, is_add_req):
             if key.startswith(GatewayState.SUBSYSTEM_PREFIX):
                 if is_add_req:
                     req = json_format.Parse(val, pb2.create_subsystem_req(), ignore_unknown_fields=True)
+                    if req.key_encrypted and req.dhchap_key:
+                        dhchap_key_to_use = None
```

Comment:
I find this more informative:
```suggestion
                        dhchap_key_decrypted = None
```

Reply:
Changed that.